src/Nox.ClientApi.Tests/Tests/Controllers/ODataResponse.cs

```diff
@@ -1,12 +1,16 @@
 ﻿
 using Newtonsoft.Json;
 
+using System.Text.Json.Serialization;
+
 namespace Nox.ClientApi.Tests.Tests.Controllers
 {
     public class ODataResponse<T>
     {
         public T Value { get; set; } = default!;
+
         [JsonProperty("@odata.count")]
+        [JsonPropertyName("@odata.count")]
```

Comment:
why do we need both? is it because System.Text? can we use newton, because its being used on the other tests and work by default better then sytem.text

or remove Newtonsoft.Json; completely